packages/ui/tsup.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from "tsup";
```

Comment:
Curiosity: why tsup over the other alternatives?

Reply:
Wondering why this configuration isn't located in packages/config

Because we're using that only here?
Couldn't we also use it to build packages/database?

Reply:
You can. I believe you'll lose the HMR if you build packages like api, auth, db instead of just transpiling them though

Reply:
We can share the config if we use tsup elsewhere.

Reply:
> Curiosity: why tsup over the other alternatives?

Fast and minimal. Vite seems overkill for this imo. Anyone else you had in mind?